### DISCUSSION TITLE
Usage of goreleaser

### DISCUSSION THREAD

**Discussion by dirien** [SchwarzIT/go-template#43] [Ideas]
Hi,

i wanted, before contributing it, preflight the idea of using goreleaser in go-template.

What are your thoughts about it?

**Comment by brumhard** [comment]
Hi,
there's already an issue about integrating goreleaser into the template #21 and of course it would be great to also release go/template itself with goreleaser. Both would be great 👍

**Reply by dirien** [reply]
I would split this then into two issues. The #21  for having a goreleaser.yaml template! 

If you dont mind, i would look to create the gorelease support for gt itself.
